contrib/scalar/scalar.c

```diff
@@ -385,21 +385,22 @@ static int stage(const char *git_dir, struct strbuf *buf, const char *path)
 	return res;
 }
 
-static int stage_file(const char *git_dir, const char *path)
+static int stage_file(const char *git_dir, const char *path, size_t skip_chars)
```

Comment:
nit: s/skip_chars/prefix_len/g

Reply:
TBH I am not _entirely_ happy about the approach I took in this patch, anyway, but I wanted to Get It Done.

A better way would be to have an optional parameter defining the directory in which the file is located under the (relative) `path`. But I only thought about this after sleeping over it... But this is just a minor nit, and it probably makes more sense to spend my time elsewhere for the moment.

Reply:
(Oh, and if it should have been unclear: this patch is in the `gvfs-helper`-related part of Scalar, therefore it won't be upstreamed, ever)